goldstone/core/models.py

```diff
@@ -78,10 +77,64 @@ class ReportData(DailyIndexDocType):
 
     INDEX_PREFIX = 'goldstone_reports-'
 
-    class Meta:
+    class Meta:          # pylint: disable=C0111,W0232,C1001
         doc_type = 'core_report'
 
 
+class EventData(DailyIndexDocType):
+    """The model for logstash events data."""
+
+    # The indexes we look for start with this string.
+    INDEX_PREFIX = 'events'
```

Comment:
should this be INDEX_PREFIX = 'events_'?  I can't recall how they are processed, but thought I saw 'logstash-' under glogging.  You may have already resolved this, but originally, I was trying to avoid undesirable matches when we had index names like goldstone-events and goldstone-metrics.  My solution was to include the separator token in the prefix.  Probably fine here, but now you have the history...